dss-api.yml

```diff
@@ -660,6 +660,258 @@ paths:
+            Timestamp of collection creation in RFC3339 format. Required to identify the collection to update.
+          required: true
+          type: string
+          format: date-time
+        - name: json_request_body
+          in: body
+          required: true
+          schema:
+            type: object
+            properties:
+              add_contents:
+                type: array
+                description: |
+                  List of new items to add to the collection. Items are de-duplicated (if an identical item is already
+                  present in the collection or given multiple times, it will only be added once).
+                items:
+                  $ref: '#/definitions/CollectionItem'
+                maxItems: 1000
+              remove_contents:
+                type: array
+                description: |
+                  List of items to remove from the collection. Items must match exactly to be removed. Items not found
+                  in the collection are ignored.
+                items:
+                  $ref: '#/definitions/CollectionItem'
+                maxItems: 1000
+              name:
+                type: string
+                description: New name for the collection.
+              description:
+                type: string
+                description: New description for the collection.
+              details:
+                type: object
+                description: New details for the collection.
+      responses:
+        200:
+          description: OK
+          schema:
+            type: object
+            properties:
+              uuid:
+                type: string
+                description: A RFC4122-compliant ID of this collection.
+              version:
+                type: string
+                description: New version of the updated collection.
+            required:
+              - uuid
+              - version
+        default:
+          description: Unexpected error
+          schema:
+            allOf:
+              - $ref: '#/definitions/Error'
+              - type: object
+                properties:
+                  code:
+                    type: string
+                    description: |
+                      Machine-readable error code.  The types of return values should not be changed lightly.
+
+                      The code `invalid_link` indicates that an item listed in the collection contents field of the
+                      input could not be resolved (either because the file, bundle, or collection does not exist on the
+                      replica specified, or because the JSON Pointer reference could not be resolved within the
+                      specified file).
+                    enum: [unhandled_exception, Forbidden, OAuthResponseProblem, not_found, invalid_link]
+                required:
+                  - code
+    delete:
+      summary: Delete a collection.
+      description: |
+        Delete a collection.
```

Comment:
I think I may have already asked this before, but I can't find the comment, but no delete-a-version?

Reply:
Right, I'd prefer to keep this simple. Not sure if I see the use case for delete-a-version on collections yet.